ui/app/com/gu/recipeasy/views/admin/dailybreakdown.scala.html

```diff
@@ -17,7 +17,7 @@
         </thead>
         <tbody>
 
-            @for( dayFigures <- dailyActivityDistribution ){
+            @for( dayFigures <- dailyActivityDistribution.reverse ){
```

Comment:
@LATaylor-guardian Finally answering this question :)

The BD code that retrieves the data is 

```
  def dailyActivityDistribution(): List[DayActivityDistribution] = {
    userEventsDates().sorted.map(date => DayActivityDistribution(date, eventsForDateAndOperationType(date, Curation).size, eventsForDateAndOperationType(date, Verification).size, eventsForDateAndOperationType(date, Confirmation).size))
  }
```
The `sorted` ensures that the `List[DayActivityDistribution]` are given in chronological order. This is fundamentally a part of the return type (at least in my mind). That being said, the UI (daily breakdown) displays them in reverse chronological order.

Because no other code than the daily breakdown use `dailyActivityDistribution()` and because no other code than `dailyActivityDistribution()` use `userEventsDates()`, I guess I could write `userEventsDates()` to return the dates in reverse chronological order, but I really don't feel doing that :) ( because the moment other functions start calling  `userEventsDates()` or `dailyActivityDistribution()` that choice will appear counter intuitive ).